sdk/iOS/src/MSOperationQueue.m

```diff
@@ -60,7 +60,7 @@ -(NSArray *) getOperationsForTable:(NSString *)table item:(NSString *)item
     MSQuery *query = [[MSQuery alloc] initWithSyncTable:syncTable];
     
     if (item) {
-        query.predicate = [NSPredicate predicateWithFormat:@"(table == %@) AND (itemId == %@)", table, item];
+        query.predicate = [NSPredicate predicateWithFormat:@"(table == %@) AND (itemId ==[c] %@)", table, item];
```

Comment:
Yes, it will need to be done.  I'm not sure what cons you see, but lets get it on the planning radar then.